### PR TITLE
Prepare images for e2e testing node-observability-operator

### PR DESCRIPTION
**Pull request by sherine-k**

### CODE REVIEW

File: ci-operator/config/openshift/node-observability-operator/openshift-node-observability-operator-main.yaml [hunk 1]
```diff
@@ -1,8 +1,46 @@
...
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
+  to: node-observability-operator
+operator:
+  bundles:
+  - as: node-observability-operator-bundle
+    base_index: operator-index
+    dockerfile_path: Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/openshift/origin-external-dns-operator:latest
```
Comment (lmzuccarelli) [inline ci-operator/config/openshift/node-observability-operator/openshift-node-observability-operator-main.yaml:37]: @sherine-k - should this not be `origin-node-observability-operator:latest`

File: ci-operator/config/openshift/node-observability-operator/openshift-node-observability-operator-main.yaml [hunk 2]
```diff
@@ -1,8 +1,46 @@
...
+    with: pipeline:external-dns-operator
```
Comment (lmzuccarelli) [inline ci-operator/config/openshift/node-observability-operator/openshift-node-observability-operator-main.yaml:38]: same here

File: ci-operator/config/openshift/node-observability-operator/openshift-node-observability-operator-main.yaml [hunk 3]
```diff
@@ -1,8 +1,46 @@
...
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
+  to: node-observability-operator
+operator:
+  bundles:
+  - as: node-observability-operator-bundle
+    base_index: operator-index
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/openshift/node-observability-operator:v0.0.1
```
Comment (lmzuccarelli) [inline ci-operator/config/openshift/node-observability-operator/openshift-node-observability-operator-main.yaml:37]: @sherine-k - This is great , I see you picked this up and it then gets substituted - nice

**Comment by lmzuccarelli** [comment]
@sherine-k - overall this looks good. We can chat later about when to take the WIP (hold) out and add a looks good to me

**Comment by bparees** [comment]
mirror names are approved
/approve

**Pull request merged**